doc/changes.rst

```diff
@@ -1 +1,3 @@
-.. include:: ../CHANGELOG.rst
```

Comment:
These seems to have deleted the include directive without replacing it at the bottom. My guess since rst uses the same prefix for a directive as a comment, this thought it was the incorrect comment, not a directive.